privacyidea_radius.pm

```diff
@@ -477,21 +482,31 @@ sub authenticate {
         &radiusd::radlog( Info, "urlparam $_ \n" ) for ( keys %params );
     }
 
-    my $ua     = LWP::UserAgent->new();
+    my $ua = LWP::UserAgent->new();
     $ua->env_proxy;
     $ua->timeout($timeout);
     &radiusd::radlog( Info, "Request timeout: $timeout " );
     # Set the user-agent to be fetched in privacyIDEA Client Application Type
     $ua->agent("FreeRADIUS");
     if ($check_ssl == false) {
         try {
-            # This is only availble with with LWP version 6
+            # This is only availble with LWP version 6
             &radiusd::radlog( Info, "Not verifying SSL certificate!" );
             $ua->ssl_opts( verify_hostname => 0, SSL_verify_mode => 0x00 );
         } catch {
             &radiusd::radlog( Error, "ssl_opts only supported with LWP 6. error: $@" );
         }
     }
+    if ($check_ssl == true) {
+        try {
+            &radiusd::radlog( Info, "Verifying SSL certificate!" );
+            &radiusd::radlog( Info, "ssl_ca_path: $SSL_CA_PATH" );
+            $ua->ssl_opts( SSL_ca_path => $SSL_CA_PATH, verify_hostname => 1 );
```

Comment:
The ssl_opts should only be set, if the SSL_CA_PATH is set, to ensure, that the system wide CAs are used, if SSL_CA_PATH is not set.
So the above if-statement should also check, if SSLA_CA_PATH is set at all.